shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/usecases/FavoritesUsecases.kt

```diff
@@ -2,29 +2,42 @@ package com.mbta.tid.mbta_app.usecases
 
 import com.mbta.tid.mbta_app.analytics.Analytics
 import com.mbta.tid.mbta_app.model.FavoriteSettings
+import com.mbta.tid.mbta_app.model.Favorites
 import com.mbta.tid.mbta_app.model.RouteStopDirection
 import com.mbta.tid.mbta_app.model.SubscriptionRequest
 import com.mbta.tid.mbta_app.repositories.IFavoritesRepository
 import com.mbta.tid.mbta_app.repositories.ISubscriptionsRepository
 import kotlin.experimental.ExperimentalObjCRefinement
 import kotlin.native.ShouldRefineInSwift
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.asStateFlow
-import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.launch
 import org.koin.core.component.KoinComponent
 
 public class FavoritesUsecases(
     private val repository: IFavoritesRepository,
     private val subscriptionsRepository: ISubscriptionsRepository,
+    private val ioDispatcher: CoroutineDispatcher,
     private val analytics: Analytics,
 ) : KoinComponent {
-    private val flow = MutableStateFlow<Map<RouteStopDirection, FavoriteSettings>?>(null)
-    public val state: StateFlow<Map<RouteStopDirection, FavoriteSettings>?> = flow.asStateFlow()
+    private val flow = MutableStateFlow<Map<RouteStopDirection, FavoriteSettings>>(emptyMap())
+    public val state: StateFlow<Map<RouteStopDirection, FavoriteSettings>> = flow.asStateFlow()
```

Comment:
```suggestion
    public val state: Flow<Map<RouteStopDirection, FavoriteSettings>> =
        repository.state.map {
            it?.routeStopDirection.orEmpty()
        }
```

This might not fix anything on its own, but I don’t think we need a second flow here just to retain the current state.

Reply:
I tried to use the same one at first but then it was crashing on init 🙃

Reply:
Would `by lazy` fix it?